pkg/config/oracle_config.go

```diff
@@ -19,6 +19,8 @@ type OracleConfiguration struct {
 
 // NeoFSConfiguration is a config for the NeoFS service.
 type NeoFSConfiguration struct {
-	Nodes   []string      `yaml:"Nodes"`
-	Timeout time.Duration `yaml:"Timeout"`
+	Nodes       []string      `yaml:"Nodes"`
+	Timeout     time.Duration `yaml:"Timeout"`
+	ContainerID string        `yaml:"ContainerID"`
+	Restore     bool          `yaml:"Restore"`
 }
```

Comment:
This service deserves a separate section in config, at the upper level of `ApplicationConfiguration`. Consider the case when Oracle service needs to be disabled (i.e. non-configured) or when you need to use different endpoints for Oracle and BlockFetcher services.